.gitmodules

```diff
@@ -28,3 +28,6 @@
 [submodule "lib/chord-metadata/chord_metadata_service"]
 	path = lib/chord-metadata/chord_metadata_service
 	url = https://github.com/c3g/chord_metadata_service.git
+[submodule "lib/opa/opa_service"]
```

Comment:
is this supposed to be lib/opa or lib/authz/opa ?

Reply:
`lib/opa`, yes. 
thank you

Reply:
never mind, we can remove this